addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -474,6 +475,57 @@ def test_mailing_shortener(self):
                     link_params=link_params,
                 )
 
+    @users('user_marketing')
+    @mute_logger('odoo.addons.mail.models.mail_mail')
+    def test_mailing_launch_flow(self):
+        """ Test that mail should be sent directly if total recipients are less than value
+        specified by config param 'mass_mailing_immediate_send_nbr', otherwise it should
+        enqueue the mailing """
+
+        # send emails immediately if total recipients are less than (or equals to)
+        # specified limit
+        mailing_1 = self.env['mailing.mailing'].create({
+            'name': 'One',
+            'subject': 'One',
+            'mailing_model_name': 'mailing.list',
+            'contact_list_ids': [(4, self.mailing_list_1.id), (4, self.mailing_list_2.id)],
+            'body_html': 'This is mass mail marketing demo'
```

Comment:
Always add a colon at the end, so it someone add a line after this one, the git history is better (check also other occurrence in this PR)
```suggestion
            'body_html': 'This is mass mail marketing demo',
```

Reply:
Ohk, Thanks for telling me! Will be careful next time.